### DISCUSSION TITLE
Is it ok to upgrade versions (Win 10 X64, 2.16.4 > 2.16.38) by simply installing over the top?

### DISCUSSION THREAD

**Discussion by TheRealSaiph** [WinMerge/winmerge#2197] [Q&A] [answered]
Probably a silly question, but I am always cautious when attempting to replace one version of a program with another. If I simply install to the same location should everything work ok? No need to uninstall first, then reinstall?

If so, it would be nice if this could be mentioned on the download page as a simple one-liner, maybe just under "The easiest way to install WinMerge is....".

Thanks in advance for any answers given.

**Comment by sdottaka** [comment, marked as answer]
If the previous version and the newly installed executable file are the same 64-bit version or 32-bit version, there should be no problem with overwriting the previous version.
Just in case, if you export `HKEY_CURRENT_USER\Software\Thingamahoochie\WinMerge` with regedit, you can restore the settings even if you uninstall if overwrite installation fails.
